src/cluster_slot_stats.c

```diff
@@ -113,6 +125,22 @@ static void addReplyOrderBy(client *c, int order_by, long limit, int desc) {
     addReplySortedSlotStats(c, slot_stats, limit);
 }
 
+/* Resets applicable slot statistics. */
+void clusterSlotStatReset(int slot) {
+    /* key-count is exempt, as it is queried separately through `countKeysInSlot()`. */
+    cluster_slot_stats[slot].cpu = 0;
+}
+
+void clusterSlotStatsReset(void) {
+    memset(cluster_slot_stats, 0, sizeof(cluster_slot_stats));
+}
+
+void clusterSlotStatsAddCpuDuration(int slot, long duration) {
```

Comment:
The API doesn't indicate if the operation was successful or not. Should we return a boolean value to indicate if it was successfully added or not?